kinto/tests/test_views_hello.py

```diff
@@ -27,3 +27,39 @@ def test_returns_bucket_id_and_url_if_authenticated(self):
         response = self.app.get('/', headers=self.headers)
         self.assertEqual(response.json['user']['bucket'],
                          '23bb0efc-e80d-829e-6757-79d41e16640f')
+
+
+class HelloViewSchemaCapabilityExposedTest(BaseWebTest, unittest.TestCase):
+
+    def get_app_settings(self, additional_settings=None):
+        settings = super(
+            HelloViewSchemaCapabilityExposedTest,
+            self).get_app_settings(additional_settings)
+        settings['experimental_collection_schema_validation'] = True
+        return settings
+
+    def test_capability_is_exposed_if_setting_is_set(self):
+        resp = self.app.get('/')
+        capabilities = resp.json['capabilities']
+        self.assertIn('schema', capabilities)
+        expected = {
+            "description": "Enforce a schema for collection records.",
+            "url": "http://kinto.readthedocs.org/en/latest/api/1.x/"
+                   "collections.html#collection-json-schema",
+        }
+        self.assertEqual(expected, capabilities['schema'])
+
+
+class HelloViewSchemaCapabilityTest(BaseWebTest, unittest.TestCase):
```

Comment:
but we need different settings for both the test cases. It has been discussed with @Natim